vpr/src/base/SetupGrid.cpp

```diff
@@ -24,6 +24,8 @@
 #include "SetupGrid.h"
 #include "vtr_expr_eval.h"
 
+#define MAX_SIZE_FACTOR 10000
```

Comment:
Look good, but also good to comment what this is here (grid size will be no more than this factor * the number of clustered block instances we expect to need).